CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 0.8.6 (Next)
+### 0.9.1 (Next)
```

Comment:
Sorry if I wasn't clear. The next version is 0.9.0, not 0.9.1, please remove the entire 0.9.1 section.